multipass-addressing/README.md

```diff
@@ -0,0 +1,52 @@
+![Precisely](../Precisely_Logo.png)
+
+# Multipass Addressing Sample
+This sample for the Spark Addressing SDK in Scala demonstrates how to improve geocoding results by performing verify first and then geocoding. With this multipass addressing example, for all first-pass results without point-level precision, a second geocoding pass is run using single line address matching, which may increase match rate.
```

Comment:
for all results without without address level precision, a second geocoding pass is run using single line input, which may increase match rate.

Reply:
ok